src/ui.h

```diff
@@ -38,7 +38,7 @@ struct uimenu_entry {
     std::string txt;      // what it says on the tin
     nc_color hotkey_color;
     nc_color text_color;
-    mvwzstr extratxt = { 0, c_unset, "", 0};
+    mvwzstr extratxt = mvwzstr { 0, c_unset, "", 0};
```

Comment:
Don't think `auto` works for [field declarations](http://stackoverflow.com/questions/11302981/c11-declaring-non-static-data-members-as-auto) unfortunately(?); this is unfortunately a verbose workaround needed until `initializer lists` are 100% supported in MSVC.

The other big hole at the moment is full `constexpr` support (mostly use inside and on class/struct declarations).